docs/2_step_by_step/2_2_experiment_and_run.md

```diff
@@ -1,11 +1,11 @@
 (experiment-and-runs)=
-# Two basic concepts: Experiment and Run
+# Two Basic Concepts: Experiment and Run
```

Comment:
```suggestion
# Two basic concepts: Experiment and Run
```
Please stick to sentence case :) 'Experiment' and 'Run' can stay upper case if you want since they are "concepts", your call I would say.
Please update all heading accordingly.